environments/production/terraform.yml

```diff
@@ -370,6 +370,18 @@ rds_instances:
       maintenance_work_mem: 960MB
       max_connections: "least({dbinstanceclassmemory/9531392},5000)"
 
+  - identifier: "pgsynclog1-production"
+    instance_type: "db.t3.xlarge"
+    storage: 20000
```

Comment:
I think this is too little, looking at the current usage: https://app.datadoghq.com/dashboard/6kr-yqd-dsy/rds-dashboard?fullscreen_end_ts=1632763658484&fullscreen_paused=false&fullscreen_section=overview&fullscreen_start_ts=1624814858484&fullscreen_widget=406592696&tpl_var_rds_instance=pgsynclog0-production&from_ts=1624814830000&to_ts=1632763630000&live=true

Reply:
Unless you think the rewrite will significantly and permanently result in lower data usage

Reply:
@dannyroberts initially, when this plan was prepared in Aug, 23rd the usage was approx 16TB. Based on that I have calculated. Haven't expected the rise in disk usage. I will plan accordingly. 

Question: 
Did we clean some data or ran something during Aug, 2nd, or Aug, 3rd, which brought down disk usage from ~29TB to ~15TB. 
![image](https://user-images.githubusercontent.com/11612356/135087236-9362fa0c-c00f-4b72-81c7-a36a3606c31e.png)

Current DB usage is also reached 20TB. 
<img width="1387" alt="image" src="https://user-images.githubusercontent.com/11612356/135087588-e94bc8c4-4074-4c6b-bd29-ec4aa8a7e07a.png">